README.rst

```diff
@@ -1,19 +1,111 @@
+####################
 edx-name-affirmation
+####################
+
+| |Build Status| |Coveralls|
+
+This library contains data model and business logic to store verified 
+personal information for learners of the edx-platform.
+This library is designed to be a pluggable library to the edx-platform.
+
+Overview
+========
+
+This library mainly stores learner's various stages of a learner's verified identifying information.
+The prevalent example is the learner's full name. To verify the full name, the learner
+has to first initate from UI that is part of the platform.
+Here are the steps of which the PII information, for example: full name, goes through:
+
+1. learner requests to update the name
+2. learner is brought to go through ID verification process
+3. learner submit the ID verification after providing supporting evidence
+4. the ID Verification reviewer approves or denies the verification
+5. the library receives the verdit and updates the name record in the model accordingly
+
+The above process can also be retriggered by a Proctored exam attempt, where the exam attempt status
```

Comment:
The process for proctoring is different than for IDV. The learner doesn't go through the IDV process, etc. It would be worth describing in more detail the name affirmation process through proctoring.